pushjack/apns.py

```diff
@@ -437,49 +437,62 @@ def sendall(self, stream, error_timeout=10):
 
 
 class APNSMessage(object):
-    """APNS message object that serializes to JSON."""
+    """
+    APNs message object that serializes to JSON.
+    Reordered and message replaced with body.
+    """
     def __init__(self,
-                 message=None,
-                 badge=None,
-                 sound=None,
-                 category=None,
-                 content_available=None,
                  title=None,
+                 body=None,
```

Comment:
This seems to be taking the place of `message`. I'd like to keep `message` as that argument name to avoid potential backwards-compatibility issues.